.github/get_results.php

```diff
@@ -100,16 +106,17 @@ function getTestResultFromFile(string $file): array
 
     return [
         'uuid' => uniqid(),
-        'title' => 'Upgrade from ' . $data[0] . ' to ' . $data[1],
-        'context' => '{"value": "Upgrade from ' . $data[0] . ' to ' . $data[1] . '"}',
+        'title' => '[' . $autoupgradeBranch . '] Upgrade from ' . $data[1] . ' to ' . $data[2],
+        'context' => '{"value": "[' . $autoupgradeBranch . '] Upgrade from ' . $data[1] . ' to ' . $data[2] . '"}',
         'skipped' => [],
         'pending' => [],
         'duration' => $duration,
         'state' => $state,
         'err' => $error,
         'date_start' => $dateStart,
         'date_end' => $dateEnd,
-        'branch' => $data[1]
+        'branch' => $data[2],
```

Comment:
Maybe it should be renamed in favor of "ps_version" or something similar if it doesn't concerned the autoupgrade, then you can change autoupgrade_branch to branch.